python_modules/automation/setup.py

```diff
@@ -14,7 +14,7 @@
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
     ],
-    packages=find_packages(exclude=["test"]),
+    packages=find_packages(exclude=["automation_tests*"]),
```

Comment:
would `*test*` work or something like that?

Reply:
`*test*` would not be good because it would eliminate any package that happened to contain the substring "test" anywhere in the name, regardless of word boundaries. This includes every subpackage of the root package. For instance if you had `dagster.core.bestest_module.ever` it would filter out `dagster.core.bestest_module` and `dagster.core.bestest_module.ever`. So it could punch random holes in the packaged code. Probably wouldn't hit anything now but not future-proof. Safest way to go is to exclude anything nested under the `<LIB_NAME>_tests` package.

BTW I think the counterintuitive (why we had it incorrect in the first place) package-finding behavior of `setuptools` is part of the reason why many projects now put their source in a root-level `src` folder or similar, more obvious how to configure it if you do that.